src/components/Charts/Charts.style.ts

```diff
@@ -21,12 +21,7 @@ function palette(props: any) {
   return props.theme.palette.chart;
 }
 
-export const ChartContainer = styled.div`
-  /* TODO(@pnavarrc): This negative margin breaks the auto-size of the chart */
-  @media (min-width: 996px) {
-    margin-left: -3rem;
```

Comment:
this negative margin was making the charts spill outside the container. what was the use case here?

Reply:
![Screen Shot 2021-06-07 at 11 57 22 AM](https://user-images.githubusercontent.com/44076375/121062883-9fcc0500-c793-11eb-9c10-bd896a3b47b0.png)

Reply:
I _think_ the idea was to align the invisible y-axis with the text content, but I don't think that helps, maybe we should just follow the spacing of the container

Reply:
also, the negative margins were already there when I wrote that, I think I wanted to get rid of them but I didn't back then

Reply:
ooo got it. yeah if you dont have objections, lets nix the negative margin